frontend/src/metabase/query_builder/components/notebook/steps/JoinStep.jsx

```diff
@@ -54,10 +54,9 @@ const stepShape = {
   revert: PropTypes.func.isRequired,
   clean: PropTypes.func.isRequired,
   actions: PropTypes.array.isRequired,
-
-  previous: stepShape,
-  next: stepShape,
 };
+stepShape.previous = stepShape;
```

Comment:
Not totally sure why this code was working fine, but after upgrading jest to version `27.5.1` it starts complaining
```
    ReferenceError: Cannot access 'stepShape' before initialization

      56 |   actions: PropTypes.array.isRequired,
      57 |
    > 58 |   previous: stepShape,
         |             ^
      59 |   next: stepShape,
      60 | };
      61 |

      at Object.<anonymous> (frontend/src/metabase/query_builder/components/notebook/steps/JoinStep.jsx:58:13)
```

[See the fail run here.](https://github.com/metabase/metabase/actions/runs/3205310802/jobs/5238005957#step:4:256)